source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -727,6 +728,16 @@ eagerly so that any operations waiting on `Connections <#connection>`_ can retry
 as soon as possible. The pool MUST NOT rely on WaitQueueTimeoutMS to clear
 requests from the WaitQueue.
 
+Clearning a load balanced pool
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A Pool MUST also have a method of clearing all `Connections <#connection>`_ for
+a specific ``serviceId`` for use when in load balancer mode. This method
```

Comment:
Could you outline what drivers are intended to do when clearing the pooling in load balanced mode _without_ a serviceId? For example, when a connection handshake fails.